detectors/GDNet/requirements.txt

```diff
@@ -1,4 +1,9 @@
 Cython
```

Comment:
Please freeze all the dependencies so that docker is up to date when the third party dependencies you use are updated